pkg/splunk/common/urls.go

```diff
@@ -0,0 +1,54 @@
+package common
+
+// URLs for Splunk Endpoints
+// List of most common endpoints used in the Splunk Operator repo
+
+// List of Splunk Enterprise APIs - Cluster Manager
+const (
+
+	//ClusterManager = "cluster-master"
+	ClusterManager = "cluster-master"
+
+	//APIClusterManagerClusterConfig = "/services/cluster/config?"
+	APIClusterManagerClusterConfig = "/services/cluster/config?"
+
+	//APICLusterManagerServices = "/services/cluster/master"
+	APICLusterManagerServices = "/services/cluster/master"
+
+	//APIClusterManagerGetInfo = "/services/cluster/master/info"
+	APIClusterManagerGetInfo = APICLusterManagerServices + "/info"
+
+	//APIClusterManagerGetPeers = "/services/cluster/master/peers"
+	APIClusterManagerGetPeers = APICLusterManagerServices + "/peers"
+
+	//APIClusterManagerRemovePeers = "/services/cluster/master/control/control/remove_peers?"
+	APIClusterManagerRemovePeers = APICLusterManagerServices + "/control/control/remove_peers?"
+
+	//APIClusterManagerApplyBundle = "/services/cluster/master/control/default/apply"
+	APIClusterManagerApplyBundle = APICLusterManagerServices + "/control/default/apply"
+
+	//APIClusterManagerGetHealth = "/services/cluster/master/health?"
+	APIClusterManagerGetHealth = APICLusterManagerServices + "/health?"
+
+	//APIClusterManagerGetSites = "/services/cluster/master/sites?"
+	APIClusterManagerGetSites = APICLusterManagerServices + "/sites?"
+
+	//APIClusterManagerGetSearchHeads = "/services/cluster/master/searchheads?"
+	APIClusterManagerGetSearchHeads = APICLusterManagerServices + "/searchheads?"
```

Comment:
For all the items listed here, I think we better name them Uri, than Api, as these two are different?

Reply:
Updated in the latest commit. Also based on our conversation we added these changes:
1) Corrected URI vs URL usage
2) Removed generic URLs from testing file to ease future usage
3) Added variable ID/counters using fmt.Sprintft() when needed.